R/table-create.R

```diff
@@ -24,6 +24,10 @@ NULL
 #' @param temporary If `TRUE`, will generate a temporary table statement.
 #' @inheritParams rownames
 #' @param ... Other arguments used by individual methods.
+#' @param pk Primary key columns.
```

Comment:
This seems too short compared to other argument names

Reply:
`primary.key`? I'm not sure this package already uses underscores for argument names :-\